pkg/install/cleanup_unix.go

```diff
@@ -209,5 +212,29 @@ func (c *CleanUpConfig) RemoveAllDirectories() error {
 	if len(msg) > 0 {
 		return fmt.Errorf("%v", strings.Join(msg, ", "))
 	}
+
 	return nil
 }
+
+func removeCNILeftovers() string {
+	var msg []string
+
+	calico10Conflist := "/etc/cni/net.d/10-calico.conflist"
+	calicoKubeconfig := "/etc/cni/net.d/calico-kubeconfig"
+	kuberouter10Conflist := "/etc/cni/net.d/10-kuberouter.conflist"
+
+	if err := os.RemoveAll(calico10Conflist); err != nil {
```

Comment:
RemoveAll removes path and any children it contains. It removes everything it can but returns the first error it encounters. If the path does not exist, RemoveAll returns nil (no error). If there is an error, it will be of type *PathError. 


I guess, that means that we going to always have error here or on line 232, depending which CNI we used to have

Reply:
Chances are that these files won't be present, so we could print debug messages here.